src/plone/api/env.py

```diff
@@ -201,6 +202,17 @@ def test_mode():
     return IS_TEST
 
 
+def read_only_mode():
+    """Returns True if you are running the zope instance on an read only ZODB.
+
+    :Example: :ref:`env_read_only_mode_example`
+    """
+    conn = Globals.DB.open()
```

Comment:
Ok, if it just open and close one it should be fine. I just wonder if this could be determined once, w/o need to do it every access. Afaik the r/o configuration cannot be changed at runtime.